pages/coc-report.vue

```diff
@@ -0,0 +1,150 @@
+
+<template>
+  <section>
+    <div class="coc-content">
+      <h1 class="coc-title">Code of Conduct - Reporting Guide</h1>
+      <div class="coc-description-container">
+        <p>
+          If you believe someone is violating <a href="/coc">the code of conduct</a>
+          we ask that you report it to the OWDDM・KWDDM organizer using the
+          <a href="mailto:owddm@proton.me">owddm@proton.me</a> email address.
+        </p>
+        <p>
+          All reports will be kept confidential. In some cases we may determine
+          that a public statement will need to be made. If that's the case, the
+          identities of all victims and reporters will remain confidential unless
+          those individuals instruct us otherwise.
+        </p>
+        <p>
+          If you believe anyone is in physical danger, please notify appropriate
+          law enforcement first.<br/>
+          If you are unsure what law enforcement agency is appropriate, please include
+          this in your report and we will attempt to notify them.
+        </p>
+        <p>
+          If you are unsure whether the incident is a violation, or whether the
+          space where it happened is covered by our code of conduct, we encourage
+          you to still report it.
+        </p>
+        <p>
+          We would much rather have a few extra reports where we decide to take
+          no action, than miss a report of an actual violation.
+        </p>
+        <p>
+          We do not look negatively on you if we find the incident is not a violation.
+        </p>
+        <p>
+          And knowing about incidents that are not violations, or happen outside
+          our spaces, can also help us to improve the Code of Conduct or the processes
+          surrounding it.
+        </p>
+        <h3>In your report please include:</h3>
+        <ul>
+          <li>Your contact info <em>(so we can get in touch with you if we need
+            to follow up)</em></li>
+          <li>Names <em>(real, nicknames, or pseudonyms)</em> of any individuals
+            involved. If there were other witnesses besides you, please try to include
+            them as well.
+          </li>
+          <li>When and where the incident occurred. Please be as specific as possible.</li>
+          <li>Your account of what occurred. If there is a publicly available record
+            (e.g. a mailing list archive or a public IRC logger) please include
```

Comment:
This seems more relevant.
```suggestion
            (e.g. a link to a Discord message or Meetup comment) please include
```

Reply:
Updated in https://github.com/owddm/owddm.github.io/pull/259/commits/0c4e8769e550e1145a4a509f84ff82b5d451495b